Dockerfile

```diff
@@ -25,7 +25,8 @@ FROM gcr.io/openssf/scorecard@sha256:8165ad910019422f40c51cbb97ff6e7db0e2e2e11fa
 
 # Build our image and update the root certs.
 # TODO: use distroless.
-FROM debian:9.5-slim
+#FROM debian:9.5-slim
```

Comment:
can you add `# debian:9.5-slim` as comment right side of the hash?

Reply:
I think if you add the tag into the image reference, dependabot can pick it up

debian:9.5-slim@sha256:ef6be890318a105f7401d0504c01c2888daa5d9e45b308cf0e45c7cb8e44634f

Reply:
also 9.5-slim is crazy old - I would use 11.2-slim or something (11 is current stable debian, 9 has been deprecated for a while)

Reply:
I wasn't aware of this possible format before and wanted to verify it in one of my repos. Looks like dependabot does the right thing and fixes up both the version and digest at the same time: https://github.com/jauderho/sandbox/pull/127/files

Nice!